### ISSUE TITLE
Horizontal scrollbar remains after reducing document width

### ISSUE THREAD

**Issue by s--g** [notepad-plus-plus/notepad-plus-plus#3538]
### Description of the Issue

When editing a document that is wider than the screen, Notepad++ introduces a horizontal scrollbar to enable access to the text that goes off the page, however if the document width is reduced (eg: by adding line breaks so that the text no longer extends past the edge of the screen), the horizontal scrollbar remains, fixed to the maximum width that the document ever was.

### Steps to Reproduce the Issue

1. Create a new document
2. Type a whole bunch of random text on the same line so that it extends past the edge of the screen
3. Observe a horizontal scrollbar appear.
4. Delete all of the text one character at a time by holding your finger on the delete or backspace key
5. Observe the horizontal scrollbar remains, fixed at the maximum width that the document ever was

### Expected Behavior

After step 4 I expect the available horizontal scroll area to scale down in size in response to the text being deleted, until there is no more text off the screen at which point the horizontal scrollbar disappears so there is no scrollbar at all just like the default case when you open a new document, or if you typed a document that never had any text going off the side of the screen.

### Actual Behavior

The horizontal scrollbar remains, and seems to remember the maximum width that the document ever was.

### Debug Information

Notepad++ v7.4.2   (32-bit)
Build time : Jun 18 2017 - 23:34:19
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Admin mode : OFF
Local Conf mode : OFF
OS : Windows 10 (64-bit)
Plugins : Explorer.dll mimeTools.dll NppConverter.dll NppExport.dll PluginManager.dll 

![after-inserting-line-breaks](https://user-images.githubusercontent.com/3196913/28305846-95ff8a28-6be0-11e7-9527-cbbe386a1c80.PNG)
![line-breaks-included](https://user-images.githubusercontent.com/3196913/28305845-95fbecf6-6be0-11e7-9cd7-6e3d97144163.PNG)

**Comment by MetaChuh** [comment]
please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.
if your issue still persists, and it is still of relevance, please feel free to file a new issue as easy to understand and reproducible as possible.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)
